website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -394,15 +538,15 @@ mysql> select * from observation_logs;
 
 Run the following command to uninstall your Kubeflow deployment:
 
-> Note: Make sure you have the correct INSTALLATION_OPTION and DEPLOYMENT_OPTION environment variables set for your chosen installation
+> Note: Make sure you have the correct INSTALLATION_OPTION, DEPLOYMENT_OPTION and PIPELINE_S3_CREDENTIAL_OPTION environment variables set for your chosen installation
 
 
 {{< tabpane persistLang=false >}}
 {{< tab header="Kustomize" lang="toml" >}}
-make delete-kubeflow INSTALLATION_OPTION=kustomize DEPLOYMENT_OPTION=rds-s3
+make delete-kubeflow INSTALLATION_OPTION=kustomize DEPLOYMENT_OPTION=rds-s3 PIPELINE_S3_CREDENTIAL_OPTION=irsa
 {{< /tab >}}
 {{< tab header="Helm" lang="yaml" >}}
-make delete-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=rds-s3
+make delete-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=rds-s3 PIPELINE_S3_CREDENTIAL_OPTION=irsa
```

Comment:
We need a separate section for creating additional profiles if using IRSA because having an IAM role is a requirement now. A separate page for it might be better so we can link to it from multiple pages terraform and cognito-rds-s3 etc.

Reply:
Can't we just link here isn't that essentially what this guide is?

Reply:
Sounds fine to use parts it but we still need documentation indicating access to S3 bucket. It will be good to use profiles guide as a reference but have a specific sample